app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -105,82 +105,120 @@
 
       <div v-if="selectedEdit == 0" class="text-left">
         <form class="space-y-5" @submit.prevent="saveEdits">
+          <!-- Grant name -->
+          <InputRow :deleteable="false" :intended="false" text="">
+            <template v-slot:label>Title:</template>
+            <template v-slot:input>
+              <BaseInput
+                v-model="form.name"
+                width="w-full"
+                placeholder="Grant name"
+                id="grant-name"
+                :rules="isDefined"
+                errorMsg="Please enter a name"
+              />
+            </template>
+          </InputRow>
+
           <!-- Owner address -->
-          <BaseInput
-            v-model="form.owner"
-            description="The owner has permission to edit the grant"
-            id="owner-address"
-            label="Owner address"
-            :rules="isValidAddress"
-            errorMsg="Please enter a valid address"
-          />
+          <InputRow :deleteable="false" :intended="false" text="has permission to edit the grant">
+            <template v-slot:label>Owner Adddress:</template>
+            <template v-slot:input>
+              <BaseInput
+                v-model="form.owner"
+                width="w-full"
+                id="owner-address"
+                placeholder="owner ethereum address"
+                :rules="isValidAddress"
+                errorMsg="Please enter a valid address"
+              />
+            </template>
+          </InputRow>
 
           <!-- Payee address -->
-          <BaseInput
-            v-model="form.payee"
-            description="The address contributions and matching funds are sent to"
-            id="payee-address"
-            label="Payee address"
-            :rules="isValidAddress"
-            errorMsg="Please enter a valid address"
-          />
-
-          <!-- Grant name -->
-          <BaseInput
-            v-model="form.name"
-            description="Your grant's name"
-            id="grant-name"
-            label="Grant name"
-            :rules="isDefined"
-            errorMsg="Please enter a name"
-          />
+          <InputRow :deleteable="false" :intended="false" text="contributions and matching funds are sent to">
```

Comment:
Current phrasing is missing a word
```suggestion
          <InputRow :deleteable="false" :intended="false" text="address contributions and matching funds are sent to">
```

Or something like "address all received funds are sent to"